src/cephadm/cephadm

```diff
@@ -5723,6 +5912,34 @@ def _get_parser():
         'gather-facts', help='gather and return host related information (JSON format)')
     parser_gather_facts.set_defaults(func=command_gather_facts)
 
+    parser_cephadmd = subparsers.add_parser(
+        'cephadmd', help='Manage cephadm running as a service')
```

Comment:
IMO. `cephadm daemon --start` looks better than `cephadm cephadmd --start`

Reply:
the --start has been dropped. if you want to rename it, why not exporter - so it's clear from the outset what the purpose is and avoids any misunderstanding?

@sebastian-philipp ^^

Reply:
Very well, let's call it exporter then!

Reply:
done